example_notebooks/transformers/utils.py

```diff
@@ -3,31 +3,55 @@
 
 
 class LLMRunner:
-    def __init__(self, model_name='google/gemma-1.1-2b-it'):
+    def __init__(self, model_name="meta-llama/Llama-3.2-1B-Instruct"):
```

Comment:
Since you change the default, shall we change it to an LLM which doesnt need extra permission? Qwen2.5-1.5B-Instruct maybe?